lib/web_ui/dev/browser_lock.yaml

```diff
@@ -1,11 +1,11 @@
 chrome:
   # It seems Chrome can't always release from the same build for all operating
   # systems, so we specify per-OS build number.
-  # Note: 741412 binary is for Chrome Version 82. Driver for Chrome version 83
+  # Note: 768962 binary is for Chrome Version 84. Driver for Chrome version 83
   # is not working with chrome.binary option.
-  Linux: 741412
-  Mac: 735194
-  Win: 735105
+  Linux: 768968

```

Comment:
These two can't be the same. Can you share the urls? I believe one of them is wrong.

Reply:
Reverted this change. This moved to https://github.com/flutter/engine/pull/20311.